APIManager.js

```diff
@@ -0,0 +1,78 @@
+class APIManager {
+    constructor() {
+        this.data = {
+            user: {
+            },
+            friends: [],
+            quote: "",
+            pokemon: {
+            },
+            meat: ""
+        }
+    }
+    
+    // gets user and friends
+    getUser() {
+        $.ajax({
+           method: "GET",
+           url: 'https://randomuser.me/api/?results=8',
+            success: (info) => {
+                this.data.user.firstName = info.results[0].name.first,
+                this.data.user.lastName = info.results[0].name.last,
+                this.data.user.city = info.results[0].location.city,
+                this.data.user.state = info.results[0].location.state,
+                this.data.user.picture = info.results[0].picture.medium
+                for(let result of info.results) {
+                    name = result.name.first + ' ' + result.name.last
+                    this.data.friends.push(name)
+                }
+                this.data.friends.splice(0, 1)
+            },
+            error: function (xhr, text, error){
+                alert(text)
+            }
+        })
+    }
+
+    getQuote() {
+        $.ajax({
+           method: "GET",
+           url: 'https://api.kanye.rest',
+            success: (info) => {
+                this.data.quote = info.quote 
+            },
+            error: function (xhr, text, error){
+                alert(text)
+            }
+        })
+    }
+
+    getPokemon() {
+        $.ajax({
```

Comment:
it would be better to define a variable up here instead
`const id = ${Math.ceil(Math.random() * 149)}`
makes your code a bit more readable and lets the reader know what kind of parameter you are sending to the api